content/en/account_management/billing/containers.md

```diff
@@ -23,6 +23,33 @@ Fargate is charged based on the concurrent number of tasks. For pricing, see the
 
 Billing of [GKE Autopilot][5] environments is the same as that of [GKE Standard][6].
 
+## Frequently asked questions
+
+**How does Datadog measure hourly on-demand usage?**
+
+Containers are metered in five minute increments. The difference is calculated between the number of containers observed and the allotment for the account. The allotment for the account is the total of the included containers (10/host for Pro and 20/host for Enterprise by default) and any contracted container commitment. The number containers exceeding the allotment are averaged out over the hour to calculate the hourly on-demand usage. This method normalizes the on-demand usage for short term spikes and variations between hosts.
```

Comment:
"The number containers exceeding the allotment are averaged out over the hour to calculate the hourly on-demand usage."

I think this sentence could be more clear. 

What if we specified that we sum each five minute increment over each hour and divided by twelve to get an average.

@ceciliawatt , thoughts on the language here?